code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,991 @@ Follow by example and make good judgement based on length which list to include
 	var/is_genetic = TRUE	// If TRUE, the marking is considered genetic and is embedded into DNA.
 	var/is_painted = FALSE	// If TRUE, the marking can be put on prosthetics/robolimbs.
 
-	tiger_stripes
-		name = "Tiger Stripes (Tajara)"
-		icon_state = "tiger"
-		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","chest","groin")
-		species_allowed = list("Tajara", "Zhan-Khazan Tajara", "M'sai Tajara")
+	aug_headplug
```

Comment:
Augments should have is_genetic set to false, so they won't come back if you are cloned.

Reply:
Good idea.

Reply:
Hrm. One catch I just realised is that certain markings can be passed off as tattoos instead, and those that are tattoos can be passed off as augments.

Reply:
Markings that are meant to be tattoos should also have their is_genetic set to false.

Reply:
chutssssssss. I wasn't thinking. You're right, tattoos wouldn't either.
But what if they're passed off as birthmarks, moles, fur, skin conditions, whatever else may be genetically influenced?